csv-validator-ui/src/main/scala/uk/gov/nationalarchives/csv/validator/ui/CsvValidatorUi.scala

```diff
@@ -148,12 +156,9 @@ object CsvValidatorUi extends SimpleSwingApplication {
     var badLines = 0
     var truncated = false
 
-    val maxNumOfLines = Try(potentialMaxNumOfLines.toInt) match {
-      case Success(number) if number > 0 => number
-      case _ =>
-        toConsole("Error: Maximum number of errors to display should be more than 0")
-        0
-    }
+    val maxCharsPerCell = convertTextboxValueToInt(potentialMaxCharsPerCell, "errors to display", toConsole)
```

Comment:
This looks correct but when I ran it, the errors were swapped (i.e. a negative value in max chars said "max number of errors should be more than 0"  and other way round. Is there a textBox / label mismatch on the UI